plugins/nanospell/plugin.js

```diff
@@ -452,6 +469,8 @@
 
 				editor.fire('SpellcheckStart');
 				editor.nanospellstarted = true;
+				self._isBusy = false;
+				self._timer = 0;
```

Comment:
This is initialized to `null`, so I'd say this should either be `null`, or the initialization should be `0`.